crates/compiler/gen_llvm/src/llvm/build.rs

```diff
@@ -3977,7 +3977,8 @@ fn expose_function_to_host_help_c_abi_v2<'a, 'ctx, 'env>(
             // In C, this is modelled as a function returning void
             (
                 &params[..],
-                &param_types[..param_types.len().saturating_sub(1)],
+                // &param_types[..param_types.len().saturating_sub(1)],
+                &param_types[..],
```

Comment:
there is something subtler going on here. We'll surely revisit this when glue is more mature

Reply:
Maybe worth leaving a comment behind? Might be hard to find again.

Reply:
no there is an assert right below